src/integrationtests/java/com/aws/greengrass/integrationtests/status/IotJobsFleetStatusServiceTest.java

```diff
@@ -162,6 +165,7 @@ void afterEach() {
 
     @Test
     void GIVEN_jobs_deployment_WHEN_deployment_finishes_THEN_status_is_uploaded_to_cloud(ExtensionContext context) throws Exception {
+        ignoreExceptionOfType(context, InvocationTargetException.class);
```

Comment:
new error?

Reply:
This is an expected exception according to me, caused due to SSL Handshake failure when communicating with cloud. And it was flaky.